### PR TITLE
removed some code smell

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description
Removed some code smell

## Changes
Removed a lot of code smell that sonar cloud was complaining about

## Related Issues
N/A

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

**Comment by J3fftw1** [comment]
want to get some feedback before i continue

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/gui/ToggleButton.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
     private final ItemStack trueTexture;
     private final ItemStack falseTexture;
     private final ToggleListener callback;
-    boolean value;
+    public boolean value;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/gui/ToggleButton.java:18]: Why would this be public and not private?

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBBlock.java [hunk 2]
```diff
@@ -509,7 +509,8 @@ public Block getAuxiliaryBlock(RelativePosition pos) {
 
     public Block getAuxiliaryBlock(Location loc, RelativePosition pos) {
         Block b = loc.getBlock();
-        int dx = 0, dz = 0;
+        int dx = 0,
+            dz = 0;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBBlock.java:513]: Variables should never be declared in pairs, seperate these declarations please.
```suggestion
        int dx = 0;
        int dz = 0;
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/gui/ToggleButton.java (hunk 1 above)
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/gui/ToggleButton.java:18]: i thought sonarcloud said it like that but will change and see if i get errors

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**